### ISSUE TITLE
 Can't import the named export 'URL' from non EcmaScript module (only default export is available)

### ISSUE THREAD

**Issue by Akash667** [frontity/frontity#868]
**_Bug report_**

- I've updated both my project and my packages to the latest Frontity versions: https://docs.frontity.org/guides/keep-frontity-updated
- I've searched for existing issues.
- I've checked the documentation: https://docs.frontity.org
- I've checked the community forum: https://community.frontity.org

**Expected behavior**
Should have installed all the dependencies 

**Observed behavior**
Frontity fails to install upon running `npx frontitty dev`

**Steps involved to reproduce the problem**
1. Use `npx frontity create my-app`
2. `cd my-app`
3. `npx frontity dev`

`ERROR in ./node_modules/node-fetch/lib/index.mjs 1157:18-21
    Can't import the named export 'URL' from non EcmaScript module (only default export is available)
     @ ./node_modules/frontity/dist/src/index.js
     @ ./node_modules/@frontity/tiny-router/src/actions.ts
     @ ./node_modules/@frontity/tiny-router/src/index.ts
     @ ./build/bundling/entry-points/server.ts`

**Info about your system**

## System:
 - OS: Windows 10 10.0.19043
 - CPU: (8) x64 Intel(R) Core(TM) i5-8300H CPU @ 2.30GHz
 - Memory: 5.47 GB / 15.85 GB
## Binaries:
 - Node: 14.17.6 - C:\Program Files\nodejs\node.EXE
 - npm: 7.5.2 - C:\Program Files\nodejs\npm.CMD
## Browsers:
 - Chrome: Not Found
 - Edge: Spartan (44.19041.1023.0), Chromium (93.0.961.52)
 - Internet Explorer: 11.0.19041.1202
## npmPackages:
 - @frontity/core: ^1.14.3 => 1.14.3
 - @frontity/html2react: ^1.7.0 => 1.7.0
 - @frontity/mars-theme: ./packages/mars-theme => 1.6.1
 - @frontity/tiny-router: ^1.4.4 => 1.4.4
 - @frontity/wp-source: ^1.11.5 => 1.11.5
 - frontity: ^1.17.0 => 1.17.0
## npmGlobalPackages:
 - frontity: Not Found
 - npx: Not Found

**Possible solution**

**Comment by TreliaGT** [comment]
Having the same issue sadly

**Comment by therealjgrant** [comment]
Same here, tried Node 14.17.6 & 16.9.1

**Comment by TreliaGT** [comment]
has something updated lately, I just started learning this today

**Comment by sainer** [comment]
I have the same problem. Node 14.16.0. I've updated both my project and my packages to the latest Frontity versions and the problem still occurs.

**Comment by luisherranz** [comment]
`node-fetch` recently changed to [ESM only](https://github.com/node-fetch/node-fetch#loading-and-configuring-the-module), maybe it is related to that. I'll take a look.

**Comment by robrecord** [comment]
This appears to be because node-fetch has a bug in version 2.6.3 and 2.6.4

I think Frontity needs to pin the version to 2.6.2

**Comment by robrecord** [comment]
@luisherranz I did explore the change to ESM only too, but it doesn't lead anywhere since we're not using version 3 in frontity.

I think this caused because `node-fetch` has a bug in version 2.6.3 and 2.6.4 where they [add an import that doesn't seem to work](https://github.com/node-fetch/node-fetch/commit/ace7536c955556be742d9910566738630cc3c2a6#r56836783).

In that case maybe frontity needs to pin the version to 2.6.2

**Comment by lluisd** [comment]
same issue here for the first time installing it

**Comment by nicholasio** [comment]
if you explicitly set the version to 2.6.2 by adding node-fetch to package.json does it solve the issue?

**Comment by luisherranz** [comment]
> if you explicitly set the version to 2.6.2 by adding node-fetch to package.json does it solve the issue?

It does indeed.

Good catch @robrecord 🙂

**Comment by lluisd** [comment]
why is closed? is still happening at me... I still have 2.6.1 version in the node_modules frontity package

**Comment by luisherranz** [comment]
Issues are closed once a fix is merged to the base branch. After that, you need to wait for the next release, which @santosguillamot started 1 hour ago: https://github.com/frontity/frontity/pull/871

**Comment by SantosGuillamot** [comment]
> After that, you need to wait for the next release, which @SantosGuillamot started 1 hour ago: #871

This is already done 🙂  The issue should be solved now.

**Comment by luisherranz** [comment]
Thanks Mario!

Please update your Frontity packages using this guide: https://docs.frontity.org/guides/keep-frontity-updated
